modules/tracker/controllers/components/ApiComponent.php

```diff
@@ -85,26 +85,24 @@ public function itemAssociate($args)
             throw new Exception('Read permission on the item required', 403);
         }
 
-        $scalarIds = explode(',', $args['scalarIds']);
+        $submissionId = $args['submissionId'];
 
-        /** @var int $scalarId */
-        foreach ($scalarIds as $scalarId) {
-            /** @var Tracker_ScalarDao $scalar */
-            $scalar = $scalarModel->load($scalarId);
+        /** @var Tracker_SubmissionDao $submission */
+        $submission = $submissionModel->load($submissionId);
 
-            if (!$scalar) {
-                throw new Exception('Invalid scalarId: '.$scalarId, 404);
-            }
-            if (!$communityModel->policyCheck(
-                $scalar->getTrend()->getProducer()->getCommunity(),
-                $user,
-                MIDAS_POLICY_ADMIN
-            )
-            ) {
-                throw new Exception('Admin permission on the community required', 403);
-            }
-            $scalarModel->associateItem($scalar, $item, $args['label']);
+        if (!$submission) {
+            throw new Exception('Invalid submission id: '.$submissionId, 404);
+        }
+
+        if (!$communityModel->policyCheck(
+            $submission->getProducer()->getCommunity(),
+            $user,
+            MIDAS_POLICY_ADMIN
```

Comment:
Do we want ADMIN here?  I realize it was here before and has been working.

`scalarAdd` takes WRITE and it seems like they should be the same.